<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportFileDialog</class>
 <widget class="QDialog" name="ExportFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导出到文件</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>121</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>过滤条件</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="allAudioRadioButton">
       <property name="text">
        <string>录制所有音频</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="allButMeAudioRadioButton">
       <property name="text">
        <string>录制他人音频</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="selectAudioRadioButton">
       <property name="text">
        <string>录制所选音频</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="selectVideoRadioButton">
       <property name="text">
        <string>录制所选视频</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="consoleTextEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>331</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>271</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>视频列表</string>
   </property>
   <widget class="QListWidget" name="videoList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>351</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="selectDirButton">
        <property name="text">
         <string>选择导出文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openDirButton">
        <property name="text">
         <string>打开导出文件夹</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>当前录制时间</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);</string>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="filterButtonGroup"/>
 </buttongroups>
</ui>
